lib/ansible/modules/web_infrastructure/ansible_tower/tower_settings.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# (c) 2018, Nikhil Jain <nikjain@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: tower_settings
+author: "Nikhil Jain (@jainnikhil30)"
+version_added: "2.7"
+short_description: Get, Modify Ansible Tower settings.
+description:
+    - Get, Modify Ansible Tower settings. See
+      U(https://www.ansible.com/tower) for an overview.
```

Comment:
this actually redirects to https://www.ansible.com/products/tower, but I don't know if that's a very stable URL either.